filer/templates/admin/filer/widgets/admin_file.html

```diff
@@ -11,24 +11,25 @@
     </div>
 
     <div class="js-dropzone dropzone" data-url="{% url 'admin:filer-ajax_upload' %}">
-        <div class="dz-default dz-message js-dropzone-message{% if object %} hidden{% endif %}" id="dropzone-message">
+        <div class="dz-default dz-message js-dropzone-message{% if object %} hidden{% endif %}" id="{{ id }}_dropzone_message">
```

Comment:
ID's are not a numbers here.
A lot of values are going from the BE like this here, so can't change it.